store/tikv/scan_test.go

```diff
@@ -107,14 +107,14 @@ func (s *testScanSuite) TestScan(c *C) {
 		}
 		err := txn.Commit(s.ctx)
 		c.Assert(err, IsNil)
-
+		mockTableID := int64(999)
```

Comment:
would you please explain why to use such a magical number?

Reply:
please leave a comment in the code.

Reply:
I think the name explains itself, it's `mocked`.

Reply:
fine

Reply:
feel free to mark it as resolved